src/main/java/co/rsk/bitcoinj/script/NonStandardErpRedeemScriptParser.java

```diff
@@ -2,28 +2,61 @@
 
 import static co.rsk.bitcoinj.script.RedeemScriptValidator.removeOpCheckMultisig;
 
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Sha256Hash;
 import co.rsk.bitcoinj.core.Utils;
 import co.rsk.bitcoinj.core.VerificationException;
 import java.util.ArrayList;
 import java.util.List;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ErpFederationRedeemScriptParser extends StandardRedeemScriptParser {
-    private static final Logger logger = LoggerFactory.getLogger(ErpFederationRedeemScriptParser.class);
+public class NonStandardErpRedeemScriptParser implements RedeemScriptParser {
```

Comment:
Shouldn't this change be in a different PR?

Reply:
Base branch updated